docs/05-UnitTests.md

```diff
@@ -181,22 +186,106 @@ To enable the database functionality in the unit tests please make sure the `Db`
+    $this->tester->dontSeeRecord('users', ['name' => 'miles']);
+}
+?>
+```
+
+The very similar approach can be used to all frameworks that have ORM implementing ActiveRecord pattern.
+These are Yii2 and Phalcon, they have methods `haveRecord`, `seeRecord`, `dontSeeRecord` working in the same manner. They also should be included with specifying `part: ORM` in order to not use functional testing actions.
+
+In case you are using Symfony2 with Doctrine you may not enable Symfony2 itself but use only Doctrine2 only:
+
+```yaml
+class_name: UnitTester
+modules:
+    enabled:
+        - Asserts
+        - Doctrine2:
+            depends: Symfony2
+        - \Helper\Unit
+```
+
+In this case you can use methods from Doctrine2 module, while Doctrine itself uses Symfony2 module to establish connection to database. In this case a test may look like:
+
+```php
+<?php
+function testUserNameCanBeChanged()
+{
+    // create a user from framework, user will be deleted after the test
+    $id = $this->tester->haveInRepository('Acme\DemoBundle\Entity\User', ['name' => 'miles']);
+    // get entity manager by accessing module
+    $em = $this->getModule('Doctrine2')->em;
+    // get real user
+    $user = $em->find('Acme\DemoBundle\Entity\User', $id);
+    $user->setName('bill');
+    $em->persist($user);
+    $em->flush();
+    $this->assertEquals('bill', $user->getName());
+    // verify data was saved using framework methods
+    $this->tester->seeInRepository('Acme\DemoBundle\Entity\User', ['name' => 'bill']);
+    $this->tester->dontSeeInRepository('Acme\DemoBundle\Entity\User', ['name' => 'miles']);
+}
+?>
+```
+
+In both examples you should not be worried about the data persistance between tests. 
+Doctrine2 module as well as Laravel4 module will clean up created data in the end of a test. 
+This is done by wrapping a test into transaction and rolling it back afterwards. 
+
 ### Accessing Module
 
 Codeception allows you to access properties and methods of all modules defined for this suite. Unlike using the UnitTester class for this purpose, using module directly grants you access to all public properties of that module.
 
-For example, if you use `Symfony2` module, here is the way you can access Symfony container:
+We already demonstrated this case in previous code piece where we accessed Entity Manager from a Doctrine2 module
+
+```php
+<?php
+/** @var Doctrine\ORM\EntityManager */
+$em = $this->getModule('Doctrine2')->em;
+?>
+```
+
+If you use `Symfony2` module, here is the way you can access Symfony container:
 
 ```php
 <?php
-/**
- * @var Symfony\Component\DependencyInjection\Container
- */
+/** @var Symfony\Component\DependencyInjection\Container */
 $container = $this->getModule('Symfony2')->container;
 ?>
 ```
 
-All public variables are listed in references for corresponding modules.
+The same can be done for all public properties of enabled module. Accessible properties are listed in module reference
```

Comment:
The same can be done for all public properties of **an** enabled module. Accessible properties are listed in **the** module reference